layouts/partials/footer.html

```diff
@@ -7,11 +7,10 @@
     <div class="row align-items-center">
     <div class="col-md-3 text-md-left text-center">
        <ul>
-        <li><a class="nav-link text-white" href="{{ site.BaseURL | relLangURL }}">HOME</a></li>
```

Comment:
Because of what we decided in #80, we still need this change. This line should not be removed.

Reply:
Thanks for that! Great to work and contribute here @jwflory .

I did some minor changes as you suggested! I switched to variable colours. from the [header.html <style> : root](https://github.com/unicef/inventory-hugo-theme/blob/main/layouts/partials/head.html#L33#L42). minor changes to previous values too from [style.css](https://github.com/unicef/inventory-hugo-theme/blob/main/assets/css/style.css)

Reply:
@yokwejuste Thanks. I'll test these changes once more since other CSS unrelated to the social media icons was changed. I'll give another pass sometime today.